optional_plugins/varianter_pict/avocado_varianter_pict/__init__.py

```diff
@@ -141,9 +141,9 @@ def __iter__(self):
         for variant in self.variants:
             base_id = "-".join([variant.get(key) for key in self.headers])
             variant_ids.append(base_id + '-' +
-                               hashlib.sha1(base_id).hexdigest()[:4])
+                               hashlib.sha1(base_id.encode()).hexdigest()[:4])
 
-        for vid, variant in itertools.izip(variant_ids, self.variants):
+        for vid, variant in zip(variant_ids, self.variants):
```

Comment:
> izip: Like zip() except that it returns an iterator instead of a list.

Yea, looks like it will not affect the code here. But since that we are not using `itertools` anymore, we should remove the import, right?

Reply:
Good point, we should! I'll (force) push a fix.

Reply:
> > izip: Like zip() except that it returns an iterator instead of a list.
>

BTW, just for completeness sake, this is true for Python 2 only. On Python3, `zip()` returns an iterator.